internal/builder/builder.go

```diff
@@ -141,6 +141,10 @@ func constructBuilder(img imgutil.Image, newName string, errOnMissingLabel bool,
 		return nil, err
 	}
 
+	if registry != "" && runImage != "" {
+		metadata.RunImages[0].Image = registry + "/" + runImage
```

Comment:
Rather than replacing the first element, should we override the whole array? So like:

`metadata.RunImages = []RunImageMetadata{{Image: "blah"}}`